### PR TITLE
LPS-165322 [Infra/Evp] Make status active and panelCategoryKey

### PR DESCRIPTION
**Pull request by TarcianaBarros**

### CODE REVIEW

**Comment by TarcianaBarros** [comment]
ci:test:sf

**Comment by TarcianaBarros** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/division.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
 		},
 		"permissions": {
 		},
+		"publish": {
+		},
```
Comment (nNilton) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/division.json:12]: Is it really necessary?

Comment (TarcianaBarros) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/division.json]: done.

**Comment by TarcianaBarros** [comment]
ci:test:sf

**Comment by TarcianaBarros** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:test:relevant

**Comment by nNilton** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by nNilton** [comment]
manually forwarded:https://github.com/brianchandotcom/liferay-portal/pull/127183

**Pull request closed without merging**